R/helpers.R

```diff
@@ -20,31 +20,23 @@ getRMNDGDPcap <- function(scenario="gdp_SSP2",
                           yearcol="year",
                           isocol,
                           valuecol="weight",
-                          isolev = F,
+                          to_aggregate = F,
                           usecache = F,
-                          gdpfile ="GDPcache.rds"){
+                          gdpCapfile){
```

Comment:
just a small note on style: arguments that are *required* (since there are no default values given), should be come before arguments with default in the argument listing.

Reply:
apart from this detail everything looks fine!

Reply:
ok, addressed in the commit below.